buf.gen.yaml

```diff
@@ -11,15 +11,19 @@ managed:
       - 'buf.build/googleapis/googleapis'
 plugins:
   - name: 'go'
+    path: ["go", "run", "google.golang.org/protobuf/cmd/protoc-gen-go"]
```

Comment:
does this make buf use the version found in `go.mod`?

Reply:
yep!